part-one/README.md

```diff
@@ -2,6 +2,8 @@
 
 In an earlier version of this book I attempted to skip any philosophical exposition and jump right into the impact of digital technology. While that may make for more gripping reading, it resulted in the verbal equivalent of building a skyscraper without a foundation: Rapid initial progress followed by total collapse.
 
-With digital technology inflating the space of what is possible, we need to establish some principles. Otherwise we will misread the trends and phenomena that are already happening today. Instead of exploring the new space for the benefit of all of humanity, we will instead try to bend it to fit the existing economic and social systems. 
+With digital technology inflating the space of what is possible, we need to establish some principles. Otherwise we will misread the trends and phenomena that are already happening today. Instead of exploring the new space of the possible for the benefit of all of humanity, we will instead try to bend it to fit existing economic and social systems.
 
-What follows is my attempt to establish a firm foundation for building a future grounded in Humanism. Much of my thinking about this has been deeply influenced by the writing of David Deutsch, in particular his book &ldquo;The Beginning of Infinity&rdquo; {{ "DEUT" | cite }}. 
+What follows is my attempt to establish a firm foundation for building a future grounded in optimism and humanism.  I illustrate why the source of optimism as a principle is the power of knowledge, and why the existence of knowledge alone provides the basis for humanism.  Much of my thinking about this has been deeply influenced by the writing of David Deutsch, in particular his book &ldquo;The Beginning of Infinity&rdquo; {{ "DEUT" | cite }}.
```

Comment:
Change that from "I illustrate why the source of optimism as a principle is the power of knowledge" to "I explain why the power of knowledge is the source of optimism as a principle"